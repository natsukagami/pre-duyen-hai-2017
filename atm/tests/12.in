541 481
6871000143871086426149204119400469264038189671584426580248944449059437736407616515335198709065429535594909180545162274685635177750110845357473177879797395997692559806972014121097287855690344592188025387676257288013582185712969998205189015288801430788275401091045542824240004325581307784325177264087218651875268415148627653137209824397370644304111242751521641694036302216691843963257135002212541686291700211916023477447757432450748066942149571916521573033410937414424607612538237451737358547945408240877582438509168587445412668027012456587451
