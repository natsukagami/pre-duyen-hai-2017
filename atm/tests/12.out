999999999999999999999999999999999999999999988827012456587451
